.gitignore

```diff
@@ -50,7 +50,7 @@ playground.xcworkspace
 # Carthage
 #
 # Add this line if you want to avoid checking in source code from Carthage dependencies.
-# Carthage/Checkouts
+Carthage/Checkouts
```

Comment:
I was surprised to see that this repo was committing Carthage checkouts.  Is it ok to remove?  I'd probably have to delete all the already committed checkouts as well to get it back in proper order, but for now, I just didn't want to commit the CRF module checkout.

Reply:
So, it works b/c mPower uses submodules. This is for the case where you *don't* use submodules and want to check in the version of the code that you are building against so you don't accidentally change things and then not have a record of it.

Reply:
Thanks for this distinction, I was building with "no submodules" flag in carthage, that may be why I was hitting weird issues, I'll try again and see if this isn't necessary anymore.

Reply:
Ah, yeah. Look at the bin/setup script for the Cartage update command. It includes using the submodules.